package PacketCore;


message Auth
{
	required int32 EncryptionKey = 1;		
}